pygfx/renderers/wgpu/shaders/imageshader.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, wobject):
             self["img_nchannels"] = 3
         elif material.map is not None:
             self["use_colormap"] = True
-            self["colorspace"] = material.map.colorspace
+            self["colorspace"] = material.map.texture.colorspace
```

Comment:
Would it be possible to add a property to `map` that is defined as:


```
@property
def colorspace(self):
    return self.texture.colorspace
```
so as to avoid this needed change in downstream users?

Perhaps even with a "warning" that tells users to use `map.texture.colorspace` instead of `map.colorspace`.

Mostly a thin transition layer.

I'm happy to make the PR.

Reply:
hmm the changes to the interface of `define_img_colormap` are more breaking for myself, so i've created a more complicated shims that work for me. 

I don't want push the focus toward these backward compatibility concerns too much.